cmd/kubeadm/app/master/selfhosted.go

```diff
@@ -340,3 +400,277 @@ func getSchedulerDeployment(cfg *kubeadmapi.MasterConfiguration, volumes []v1.Vo
 func buildStaticManifestFilepath(name string) string {
 	return path.Join(kubeadmapi.GlobalEnvParams.KubernetesDir, "manifests", name+".yaml")
 }
+
+func apiServerProjectedVolume() v1.Volume {
```

Comment:
@jbeda do we have a roadmap towards mo-betta securing secrets?  

We should prioritize this as a SIG, b/c it's important to the HA story.